nodeinstaller/internal/constants/testdata/expected-configuration-qemu-snp.toml

```diff
@@ -12,8 +12,9 @@ default_vcpus = 1
 disable_block_device_use = false
 disable_guest_selinux = true
 disable_image_nvdimm = true
+disable_nesting_checks = true
 disable_selinux = false
-enable_annotations = ['enable_iommu', 'virtio_fs_extra_args', 'kernel_params', 'default_vcpus', 'default_memory']
```

Comment:
discussion needed

Reply:
I don't think these are strictly required right now, adding cpus doesn't work anyway, memory can be controlled via resourceRequest. We should revisit and align this in the future, but we can keep it for now.

Reply:
related upstream: https://github.com/kata-containers/kata-containers/commit/abf52420a4cd94bdc688a11f8af52885bd85d8b5